common/components/componentsBySection/FindProjects/ProjectSearchContainer.jsx

```diff
@@ -1,10 +1,10 @@
+//TODO: delete this filf if we choose to remove alerts button. the file is unused right now.
```

Comment:
Actually I see that Alerts button has been removed.  Did we make the decision to remove it entirely?

Reply:
It's been removed from the design but I don't believe we've made a concrete decision one way or the other on deleting it, so I want to leave that todo and file until I get a yes/no.